examples/terraform/bootkube-install/README.md

```diff
@@ -35,8 +35,20 @@ matchbox_rpc_endpoint = "matchbox.example.com:8081"
 ssh_authorized_key = "ADD ME"
 ```
 
+##### Self-hosted etcd
+
 You may set `experimental_self_hosted_etcd = "true"` to deploy "self-hosted" etcd atop Kubernetes instead of running etcd on hosts directly. Warning, this is experimental and potentially dangerous.
 
+##### OEM Customisation
+
+You may set `container_linux_oem = "<oem_id>"` to use a specific oem version as installation base. For example setting `container_linux_oem = "vmware_raw"` will use an installation image (as base) with included Guest Tools.
+
+Popular OEM images: `ami`, `cloudstack`, `digitalocean`, `packet`, `rackspace`, `vmware_raw` and `xen`.
```

Comment:
I'm using VMware's platform, but i listed all possible images (at the time of writing) as they maybe of use to someone experimenting.